src/js/components/stock-movement-wizard/outbound/PickPage.jsx

```diff
@@ -906,7 +951,7 @@ class PickPage extends Component {
                   </a>
                   <button
                     type="button"
-                    onClick={() => this.clearPicklist()}
+                    onClick={() => this.confirmClearPicklist()}
```

Comment:
@alannadolny this might be a shortcut, but I believe I explained once, that I don't like passing arguments that are not used in a method.
Your suggestion would make this evaluate to:
```js
onClick={(e) => this.confirmClearPicklist(e)}
```
even though we do not expect to pass any param to this method.
Why I remember this is that once, I gave you the opposite suggestion in a PR :laughing: